NEWS.md

```diff
@@ -3,6 +3,8 @@ Unreleased
 * Remove `suspenders` system executable
 * Introduce `suspenders:accessibility` generator
 * Introduce `Suspenders::Generators::APIAppUnsupported` module and concern
+* Introduce `suspenders:factories` generator
+* Introduce `Suspenders::Generators::Helpers` module
```

Comment:
I read NEWS.md as an end user curious about what an upgrade gets me. Is the `Helpers` module something an end user would be interested in?

Reply:
That's a good distinction. Addressed here: 7c84f1d442a3120d6af2b721a10a5c2ad4ac1938

I'll create a separate commit to remove `Suspenders::Generators::APIAppUnsupported`